examples/docker.yml

```diff
@@ -2,9 +2,9 @@ kernel:
   image: "linuxkit/kernel:4.9.x"
   cmdline: "console=ttyS0 console=tty0 page_poison=1"
 init:
-  - linuxkit/init:63eed9ca7a09d2ce4c0c5e7238ac005fa44f564b
+  - linuxkit/init:deea956a9ab07bf262083e93a86930bdc610cc2fb
```

Comment:
the `b` here is wrong. The tag on hub is `deea956a9ab07bf262083e93a86930bdc610cc2f`